code-examples/quarkus-service-registry-quickstart/producer/src/main/resources/application.properties

```diff
@@ -3,30 +3,29 @@
 # Configure the outgoing `quotes` Kafka topic
 mp.messaging.outgoing.quotes.apicurio.registry.auto-register=true
 mp.messaging.outgoing.quotes.apicurio.registry.find-latest=true
-mp.messaging.outgoing.quotes.apicurio.registry.artifact-resolver-strategy=io.apicurio.registry.serde.avro.strategy.TopicRecordIdStrategy
+mp.messaging.outgoing.quotes.apicurio.registry.artifact-resolver-strategy=io.apicurio.registry.serde.strategy.TopicIdStrategy
 mp.messaging.outgoing.quotes.apicurio.registry.avro-datum-provider=io.apicurio.registry.serde.avro.ReflectAvroDatumProvider
 mp.messaging.outgoing.quotes.connector=smallrye-kafka
 mp.messaging.outgoing.quotes.value.serializer=io.apicurio.registry.serde.avro.AvroKafkaSerializer
 mp.messaging.outgoing.quotes.key.serializer=org.apache.kafka.common.serialization.StringSerializer
 mp.messaging.outgoing.quotes.merge=true
 
-%dev.mp.messaging.outgoing.quotes.apicurio.auth.realm=${OAUTH_REALM:rhoas}
-%dev.mp.messaging.outgoing.quotes.apicurio.auth.service.url=${OAUTH_SERVER_URL:https://identity.api.openshift.com/auth}
-%dev.mp.messaging.outgoing.quotes.apicurio.auth.client.id=${CLIENT_ID}
-%dev.mp.messaging.outgoing.quotes.apicurio.auth.client.secret=${CLIENT_SECRET}
+%dev.mp.messaging.outgoing.quotes.apicurio.auth.service.token.endpoint=${RHOAS_OAUTH_TOKEN_URL:https://identity.api.openshift.com/auth/realms/rhoas/protocol/openid-connect/token}
+%dev.mp.messaging.outgoing.quotes.apicurio.auth.client.id=${RHOAS_CLIENT_ID}
+%dev.mp.messaging.outgoing.quotes.apicurio.auth.client.secret=${RHOAS_CLIENT_SECRET}
 
-mp.messaging.connector.smallrye-kafka.apicurio.registry.url=${REGISTRY_URL}
+mp.messaging.connector.smallrye-kafka.apicurio.registry.url=${SERVICE_REGISTRY_URL}
```

Comment:
Missing Env var. We should have

mp.messaging.connector.smallrye-kafka.apicurio.registry.url=${SERVICE_REGISTRY_URL}${SERVICE_REGISTRY_CORE_PATH

Reply:
Doing it right away!